algorithms/dqn/networks.py

```diff
@@ -26,6 +27,58 @@ def get_dist_q(self, x: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
         dist, q = self.fc_layers.get_dist_q(x)
         return dist, q
 
+    def reset_noise(self):
+        self.fc_layers.reset_noise()
+
+
+class NoisyLinear(nn.Module):
```

Comment:
Should add descriptions

Reply:
class NoisyWrapper(nn.Module):
    def __init__(self, std_init=0.5):
        self.std_init = std_init

    def _call_(self, in_size: int, out_size: int):
        self.noisy_layer = NoisyNet(in_size, out_size, self.std_init)

    def forward(self, x: torch.tensor) -> torch.Tensor:
        reutn self.noisy_layer(x)